recipes/install.rb

```diff
@@ -12,6 +12,7 @@
   gpgkey node['rundeck_server']['yum']['gpgkey']
   enabled node['rundeck_server']['yum']['enabled']
   action node['rundeck_server']['yum']['action']
+  options node['rundeck_server']['yum']['options']
```

Comment:
please ensure you are not passing `nil` to the property by adding `unless node['rundeck_server']['yum']['options'].nil?` at the end

Reply:
Thanks for the comment. I updated the PR.